src/main/webapp/about.jsp

```diff
@@ -1,3 +1,4 @@
+<%@ page pageEncoding="UTF-8" trimDirectiveWhitespaces="true" %>
```

Comment:
While this should work, existing directives of this kind also has `language="java" contentType="text/html; charset=UTF-8"`. Let's standardize; either everything has those two as well, or everything does not.